General improvements

waspper opened the pull request.

Sergey-Orlov commented:
Also please add changes for this comment https://github.com/skilld-labs/entity_relationship/pull/3#discussion_r398024572 from Andy

andypost commented:
Also would be nice to check grammar like https://www.drupal.org/project/drupal/issues/2972224

waspper commented:
Updated. Added change suggested at https://github.com/skilld-labs/entity_relationship/pull/3#discussion_r398024572, and some changes related to `cspell`.
